ibis/expr/types/core.py

```diff
@@ -24,6 +24,10 @@
     from .generic import ValueExpr
 
 
+# TODO(kszucs): value exprs should expose a dtype() function
+# TODO(kszucs): table exprs should expose a schema() function
```

Comment:
`TableExpr` already exposes a `schema` method, and `ValueExpr` has the `type()` method.

Reply:
Removed comments.